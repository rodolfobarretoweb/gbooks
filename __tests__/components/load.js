import React from 'react';
import Load from '../../src/components/load';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Load show/>).toJSON();
  expect(tree).toMatchSnapshot();
});
