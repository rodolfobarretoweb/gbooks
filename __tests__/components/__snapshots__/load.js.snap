// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="progress"
>
  <div
    aria-valuemax={100}
    aria-valuemin={0}
    aria-valuenow={100}
    className="progress-bar active progress-bar-striped"
    role="progressbar"
    style={
      Object {
        "width": "100%",
      }
    }
  />
</div>
`;
